app/models.py

```diff
@@ -1385,8 +1389,6 @@ def is_readable_by(self, user_or_token):
         """
         return user_or_token.id in [self.created_by_id, self.id]
 
-    __table_args__ = (sa.UniqueConstraint("created_by_id", "name", name="token_name_userid_unique_constraint"),)
```

Comment:
reinstate: we need to create a unique constraint on id+name.
See https://github.com/cesium-ml/baselayer/pull/262/files